### PR TITLE
Issue1608 finalize single zone VAV models

### PR DESCRIPTION
**Pull request by kuzha**
@mwetter 
This PR is to address the issues listed in the issue 1608. In addition, the following major changes have been made:
- Removed the [BaseClasses.InternalLoads](https://github.com/lbl-srg/modelica-buildings/blob/master/Buildings/Air/Systems/SingleZone/VAV/Examples/BaseClasses/InternalLoads.mo) block;
- Exposed parameters of PID controllers in the composite controller [ChillerDXHeatingEconomizerController](https://github.com/lbl-srg/modelica-buildings/blob/master/Buildings/Air/Systems/SingleZone/VAV/ChillerDXHeatingEconomizerController.mo) (the same way as in the [Guideline36 controller](https://github.com/lbl-srg/modelica-buildings/blob/master/Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/SingleZone/VAV/Controller.mo));
- Changed the VAV controller from P control to PI control in the [Guideline36 example model](https://github.com/lbl-srg/modelica-buildings/blob/master/Buildings/Air/Systems/SingleZone/VAV/Examples/Guideline36.mo) to avoid chattering due to operation mode switch;
- Added missing instance comments, default names of blocks, final statements etc.

This branch merged the latest master branch as well as the branch issue1893 with the latest updates from [PR2017](https://github.com/lbl-srg/modelica-buildings/pull/2017) in order to test the switches of operation modes in the simulation.
The remote lbl-srg:issue1608_compareSZVAV branch should also have these two updates first in order to see the real commits and changed files.

### CODE REVIEW

File: Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerHeatingFan.mo [hunk 1]
```diff
@@ -1,71 +1,115 @@
...
+      enable=controllerTypeFan == Buildings.Controls.OBC.CDL.Types.SimpleController.PD
+          or controllerTypeFan == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
   parameter Real minAirFlo(
-    min=0,
-    max=1,
-    unit="1")
+    final min=0,
+    final max=1,
+    final unit="1")
     "Minimum airflow rate of system";
```
Comment (JayHuLBL) [inline Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerHeatingFan.mo:51]: Better to change the comment to `Minimum airflow fraction of system". This is less confused and consistent with the one in `Buildings.Air.Systems.SingleZone.VAV.ChillerDXHeatingEconomizerController`.

File: Buildings/Air/Systems/SingleZone/VAV/ChillerDXHeatingEconomizerController.mo [hunk 2]
```diff
@@ -6,166 +6,248 @@ model ChillerDXHeatingEconomizerController
...
-    "Gain of controller for fan"
-    annotation(Dialog(group="Control gain"));
-
-  parameter Real kEco(min=Modelica.Constants.small) = 4
-    "Gain of controller for economizer"
-    annotation(Dialog(group="Control gain"));
+  parameter Buildings.Controls.OBC.CDL.Types.SimpleController controllerTypeHea=
+    Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller"
+    annotation(Dialog(group="Heating coil signal"));
```
Comment (JayHuLBL) [inline Buildings/Air/Systems/SingleZone/VAV/ChillerDXHeatingEconomizerController.mo:23]: For the parameter group name, I suggest to use `Setpoints`, `Heating coil control`, `Cooling coil control`, `Fan control`, and `Economizer control`.

**Comment by JayHuLBL** [comment]
@kuzha Please see inline comments. I also noticed that the issue #1608 includes 

- [x] Relevant controller gains
- Minimum OA and system airflows
- Allowable supply air temperature limits
- Zone CO2 monitoring

I see that the 1st item has been addressed, how about other three items?

File: Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerHeatingFan.mo (hunk 1 above)
Comment (kuzha) [reply Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerHeatingFan.mo]: accepted your suggestion

File: Buildings/Air/Systems/SingleZone/VAV/ChillerDXHeatingEconomizerController.mo (hunk 2 above)
Comment (kuzha) [reply Buildings/Air/Systems/SingleZone/VAV/ChillerDXHeatingEconomizerController.mo]: changed the group names as you suggested, also for g36 controller

**Comment by kuzha** [comment]
@JayHuLBL  

The following three items has been addressed previously:

 For Minimum OA and system airflows, this [commit](https://github.com/lbl-srg/modelica-buildings/commit/c4abcd8ab205e7fe16282133950f31c1149d3cff) makes the two models comparable;
 For supply air temperature limits, this [commit](https://github.com/lbl-srg/modelica-buildings/commit/7d6313236faebd0cf598aa033a7d85bcb63e24d1) makes two models the same SAT limits;
 For Zone CO2 monitoring, please see this [commit](https://github.com/lbl-srg/modelica-buildings/commit/b64dc9304db55de17b054f7b43bc7eed37d87c9f).

**Pull request merged**